decidim-core/db/migrate/20181204110723_remove_following_users_count_from_users.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class RemoveFollowingUsersCountFromUsers < ActiveRecord::Migration[5.2]
+  def up
+    remove_column :decidim_users, :following_users_count
```

Comment:
Should we also reset the following counter since it was only increasing when a user was followed?